artemis-core-client/src/main/java/org/apache/activemq/artemis/core/protocol/core/impl/RemotingConnectionImpl.java

```diff
@@ -387,4 +388,15 @@ public void setClientID(String cID) {
    public String getClientID() {
       return clientID;
    }
+
+   @Override
+   public void killMessage(SimpleString nodeID) throws ActiveMQException {
+      if (clientVersion < DisconnectConsumerWithKillMessage.VERSION_INTRODUCED) {
+         throw new ActiveMQException(String.valueOf(clientVersion));
```

Comment:
do not throw anything.. simply ignore it and close the client accordingly. no semantic change should be made